hudi-client/src/test/java/org/apache/hudi/TestWriteStatus.java

```diff
@@ -37,6 +38,14 @@ public void testFailureFraction() {
     assertTrue(status.getFailedRecords().size() > 0);
     assertTrue(status.getFailedRecords().size() < 150); // 150 instead of 100, to prevent flaky test
     assertTrue(status.hasErrors());
+
+    assertFalse(status.hasGlobalError());
+    status.setGlobalError(t);
+    assertTrue(status.hasGlobalError());
+    assertEquals(status.getGlobalError(), t);
```

Comment:
IMO, testing a pure getter/setter method of a field is not necessary.

Reply:
@prashantwason I agree with @yanghua, unless there is a method (get, has..etc) which is performing calculations to return the value, testing simply the getter is of no real use. I understand it increased the coverage of the system and if there is no real business logic to test, it is ok to leave it. We can then focus on adding cases for other classes where there is untested business logic.

Reply:
ok.